uint8 PWM_duty
uint8 PWM_freq
uint8 PWM_valve_nr
